elm.json

```diff
@@ -12,6 +12,7 @@
         "elm/core": "1.0.0 <= v < 2.0.0"
     },
     "test-dependencies": {
-        "elm-explorations/test": "1.2.2 <= v < 2.0.0"
+        "elm-explorations/test": "1.2.2 <= v < 2.0.0",
```

Comment:
I'm aware this PR is already doing stuff far above just adding two functions, but I just wanted to note that this could be "2.0.1 <= v < 3.0.0" now, if we upgrade the `elm-test` npm package as well :)

Reply:
Fixed, thank you!